.github/workflows/pre-release.yml

```diff
@@ -0,0 +1,55 @@
+name: Pre-Release
```

Comment:
This config creates a new pre-release for every branch that is not master?

Reply:
Yes